charmcraft/templates/init-simple/charmcraft.yaml.j2

```diff
@@ -1,5 +1,9 @@
+# Copyright {{ year }} {{ author }}
+# See LICENSE file for licensing details.
+
 # Learn more about charmcraft.yaml configuration at:
 # https://juju.is/docs/sdk/charmcraft-config
+#
```

Comment:
Some updated comment wording from Daniele in my branch: https://github.com/canonical/template-operator/blob/charmcraft-templates/default/charmcraft.yaml